src/main/java/org/sagebionetworks/bridge/validators/SharedModuleMetadataValidator.java

```diff
@@ -138,6 +139,15 @@ public void validate(Object target, Errors errors) {
             if (metadata.getVersion() <= 0) {
                 errors.rejectValue("version", "can't be zero or negative");
             }
+            
+            // tags
+            if (metadata.getTags() != null) {
+                for (String tag : metadata.getTags()) {
+                    if (tag.length() > TAG_MAX_LENGTH) {
+                        errors.rejectValue("tag", "can't be more than " + TAG_MAX_LENGTH + " characters");
+                    }
```

Comment:
The field value has to be "tags[I]" as tags is a List.

Reply:
Would "tags["+tag+"]" make sense for this? getTags() returns a set.

Reply:
You're right on the server it's a set, so tags[foo] would be the key